src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/inScalar.txt

```diff
@@ -34,21 +34,19 @@ true
 >> "" in ""
 true
 
-
-// If Blank is on either side, operator is false. 
-// Use If() to coerce Blank to string. 
->> Blank() in Blank()
-false
+// Blank values are coerced to strings
```

Comment:
I noticed that "" in Blank() and Blank() in "" both produce binding errors that seem related to the tabular in.  Will this fix affect those? We should have explicit tests for those either way.

Reply:
The operator `in` is overloaded - it can be used both for "string in string" or "record in table". Since Blank() doesn't have any type information, the binder doesn't know exactly which overload to take, and it will pick one - probably the incorrect one. This is not a real scenario - blank values would typically come from unitialized variables, or from a typed expression (such as `If(1<0,"hello")`, which is why I'm using the `If(1<0,<typed expression>)` syntax to specify a "typed blank value"